tests/tape/tapes/test_cv_param_shift.py

```diff
@@ -655,11 +656,11 @@ def test_interferometer(self, t, tol):
 
 
 class TestVarianceQuantumGradients:
-    """Tests for the quantum gradients of various gates
-    with variance measurements"""
+    """Tests for the quantum gradients of qml.various gates
+    with qml.variance measurements"""

```

Comment:
```suggestion
    """Tests for the quantum gradients of various gates
    with variance measurements"""
```
`qml.various()` is my favourite so far :rofl:

Reply:
Oh no 🤕